tools/config/Tomb1Main_ConfigTool/Models/Specification/Specification.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+using Tomb1Main_ConfigTool.Utils;
+
+namespace Tomb1Main_ConfigTool.Models;
+
+public class Specification
+{
+    public Dictionary<string, List<EnumOption>> Enums { get; private set; }
+    public List<Category> CategorisedProperties { get; private set; }
+    public List<BaseProperty> Properties { get; private set; }
+
+    public Specification(string sourceData)
+    {
+        JObject data = JObject.Parse(sourceData);
+        Dictionary<string, List<string>> enumTypes = JsonConvert.DeserializeObject<Dictionary<string, List<string>>>(data[nameof(Enums)].ToString());
```

Comment:
The lines seem to be a bit long, can you run it through some autoformatter? (Optional, best C# autoformatters are proprietary AFAIR)

Reply:
Updated in [87bf6ae](https://github.com/rr-/Tomb1Main/pull/637/commits/87bf6ae5de281eb53de6382941738394d8b1b2f6).
This should be a bit more readable now. Unfortunately the second `JsonConvert.DeserializeObject` call has to remain as it requires the custom property converter, whereas the enums can be parsed more simply with `JObject`.